src/Domain/Entity/Unfetched/UnfetchedProgrammeItem.php

```diff
@@ -25,7 +26,8 @@ public function __construct()
             false,
             0,
             MediaTypeEnum::UNKNOWN,
-            0
+            0,
+            new Options()
```

Comment:
Should be UnfetchedOptions, as we don't know what the options are, rather than "we know them any they're empty"